mne/epochs.py

```diff
@@ -675,6 +675,9 @@ def average(self, picks=None):
         Computes an average of all epochs in the instance, even if
         they correspond to different conditions. To average by condition,
         do ``epochs[condition].average()`` for each condition separately.
+
+        Will not include ICA channels, when picks=None. If picks = None and the
```

Comment:
Thanks @mmagnuski 

I should have nested under `if picks is None:` to begin with. Thanks for the catch. 

I also like your docstring note better. Much clearer, I added it with only a few changes.